app/guid-user/quickfiles/template.hbs

```diff
@@ -1,5 +1,16 @@
 {{title (t 'quickfiles.title' user-name=user.fullName)}}
 <div class='container'>
+    <span id="drp" local-class="OsfDaterange" >
+        {{date-range-picker
+            label='Show analytics for date range:'
```

Comment:
Probably want to make this an i18n string and do `label=(t  'whatever.key')`

Reply:
Oh, yeah, for sure. If this were intended to be real code that were going to go into the codebase, absolutely.

Reply:
You can make it that way--I would just add the dependency and document the changes for using it in the PR description for reference later when we get the analytics page. Looks like it's what we need.